app/src/main/java/com/example/una/Challenge.java

```diff
@@ -0,0 +1,27 @@
+package com.example.una;
+
+public class Challenge {
+    String challengeTitle;
+    String challengeDescription;
+
+    public Challenge(String challengeTitle, String challengeDescription) {
```

Comment:
if you have a constructor to set the title and description at the same time, do you need the setters for individual fields? I don't see them being modified independently